docs/source/table.rst

```diff
@@ -129,6 +129,15 @@
      - 252
      - :ikala:`\ `
 
+   * - Haydn op20
+     - - scores: ✅
+       - annotations: ✅
```

Comment:
Does this dataset have `audio`, `midi` or something similar? For consistency add either even if you have to indicate it's not available maybe?

Reply:
done